### PR TITLE
add warning logs when updating builder and bindings

### PR DESCRIPTION
**Pull request by smile37773**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
spring

File: src/spring/azext_spring/commands.py [hunk 1]
```diff
@@ -299,19 +299,27 @@ def load_command_table(self, _):
...
-        g.custom_command('create', 'create_or_update_buildpack_binding')
-        g.custom_command('set', 'create_or_update_buildpack_binding')
+        g.custom_command('create', 'create_or_update_buildpack_binding', confirmation='Creating bindings will regenerate images for all app deployments using this builder.' +
+                         'These new images will be used after app restart either manually by yourself or automatically by Azure Spring Apps in regular maintenance tasks. ' +
+                         'Use CLI command --"az spring build-service builder show-deployments" to view the app deployment list of the builder. Do you want to continue?')
+        g.custom_command('set', 'create_or_update_buildpack_binding', confirmation='Setting bindings will regenerate images for all app deployments using this builder.' +
+                         'These new images will be used after app restart either manually by yourself or automatically by Azure Spring Apps in regular maintenance tasks. ' +
+                         'Use CLI command --"az spring build-service builder show-deployments" to view the app deployment list of the builder. Do you want to continue?')
```
Comment (zhoxing-ms) [inline src/spring/azext_spring/commands.py:317]: Adding confirmation steps to existing commands may cause breaking changes to user scripts that have already used them

Comment (smile37773) [reply src/spring/azext_spring/commands.py]: OK, updated them to warning logs.

**Comment by yonzhan** [comment]
Please fix the conflicting files

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**